database/src/migrations/20210107132219_setup_biohub_db.ts

```diff
@@ -38,7 +38,7 @@ export async function up(knex: Knex): Promise<void> {
   );
 
   await knex.raw(`
-    REVOKE ALL PRIVILEGES ON SCHEMA public FROM PUBLIC;
+    REVOKE CREATE ON SCHEMA public FROM PUBLIC;
```

Comment:
keep in mind that this wont actually run in dev, unless we blow the dev pods away and remake them (which we can do for now, since we dont have any data we care about losing).

Reply:
how do we do this?